mbs_results/unsorted/selective_editing.py

```diff
@@ -91,9 +91,8 @@ def create_standardising_factor(
         each reference.
 
     """
-    previous_df = dataframe[(dataframe[period] == previous_period)]
-    previous_df = previous_df[previous_df[question_code].isin([40, 49])]
-
+    previous_df = dataframe[(dataframe[period] == period_selected)]
+    previous_df = previous_df[previous_df[question_no].isin([40, 49])]
```

Comment:
Is it going to be always 40 49 ? If so could add this on top as a constant to be clear it won't be changed